python/xir/program.py

```diff
@@ -30,6 +30,8 @@ def get_floats(params: Union[List, Dict]) -> Union[List, Dict]:
 
     return params_with_floats
 
+Wire = Union[int, str]
```

Comment:
Love it! 💯

Any thoughts on making a similar alias for parameters?  I think it would look something like this:
```python
Params = Union[List[float], Dict[str, float]]
```

Reply:
That's a good idea i think!